RoboFile.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * This is project's console commands configuration for Robo task runner.
+ *
+ * @see  http://robo.li/
+ */
+
+require __DIR__ . '/vendor/autoload.php';
+
+
+class RoboFile extends \Robo\Tasks
```

Comment:
Yes, for now just bash but I'm planning on moving bash scripts to Robo later on,